### ISSUE TITLE
Robust Annotation of Mobile Application Interfaces in Methods for Accessibility Repair and Enhancement

### ISSUE THREAD

**Issue by Wotipati** [Wotipati/Blind_Accessibility_Papers#76]
### Links
- Paper: https://dl.acm.org/citation.cfm?id=3242616

### Abstract
- Accessibility issues in mobile apps make those apps difficult or impossible to access for many people. Examples include elements that fail to provide alternative text for a screen reader, navigation orders that are difficult, or custom widgets that leave key functionality inaccessible. Social annotation techniques have demonstrated compelling approaches to such accessibility concerns in the web, but have been difficult to apply in mobile apps because of the challenges of robustly annotating interfaces. This research develops methods for robust annotation of mobile app interface elements. Designed for use in runtime interface modification, our methods are based in screen identifiers, element identifiers, and screen equivalence heuristics. We implement initial developer tools for annotating mobile app accessibility metadata, evaluate our current screen equivalence heuristics in a dataset of 2038 screens collected from 50 mobile apps, present three case studies implementing runtime repair of common accessibility issues, and examine repair of real-world accessibility issues in 26 apps. These contributions overall demonstrate strong opportunities for social annotation in mobile accessibility.

- モバイルアプリをアクセシブルにするためのアノテーションツールの提案
- キーとなる画面のスクリーンショット上でアクセシビリティ用のメタデータを編集することが可能
- 50個のアプリ（2038画面）のデータセットを利用して評価

<img width="1033" alt="スクリーンショット 2019-10-30 16 02 40" src="https://user-images.githubusercontent.com/26996041/67836488-158bdb80-fb30-11e9-95bd-abd8679cd613.png">
